### PR TITLE
Fix/home 72 text layout improve

### PR DESCRIPTION
**Pull request by Phabes**

### CODE REVIEW

File: .storybook/stories/AlertBanner/AlertBanner.stories.tsx [hunk 1]
```diff
@@ -23,39 +23,26 @@ const MyAlertBanner = {
...
-  const [open, setOpen] = useState(true);
-
-  return (
-    <AlertBanner
-      text={'Changes saved'}
-      isOpen={open}
-      onClose={() => setOpen(false)}
-    />
-  );
+  return <AlertBanner text={'Changes saved'}
```
Comment (mateusz-kleszcz) [inline .storybook/stories/AlertBanner/AlertBanner.stories.tsx:26]:
```suggestion
  return <AlertBanner text="Changes saved"/>
```

File: redux/devicesSlice.ts [hunk 2]
```diff
@@ -58,8 +59,8 @@ export const devicesSlice = createSlice({
       );
     },
     clearDeviceState: state => {
-      state.devicesList = [];
-      state.isLoading = true;
+      // state.devicesList = [];
+      // state.isLoading = true;
```
Comment (mateusz-kleszcz) [inline redux/devicesSlice.ts:63]: Revert changes before merge

File: views/BuzzerView/BuzzerView.tsx [hunk 3]
```diff
@@ -24,7 +24,12 @@ export const BuzzerView: FC<BuzzerViewProps> = ({ route }) => {
 
   return (
     <LayoutProvider
-      navbar={<Navbar text={`${location} - ${nodeId} - ${buzzer?.name}`} />}>
+      navbar={
+        <Navbar
+          text={`${location} - ${nodeId} - ${buzzer?.name}`}
+          variant="header-small"
```
Comment (mateusz-kleszcz) [inline views/BuzzerView/BuzzerView.tsx:30]: I'm not fond of inconsistent navbar font size, it should be the same in all views. Please apply it everywhere as `header-small`

File: views/DeviceListView/components/controls/LightControls.tsx [hunk 4]
```diff
@@ -78,40 +80,44 @@ export const LightControls: FC<AdditionalControlsProps> = ({
...
-      />
-      <Button
-        text="Wyłącz w danym nodzie"
-        variant="error"
-        hasFullWidth
-        onPress={handleNodeLightsOff}
-      />
-    </>
+    <View style={styles.container}>
+      <View style={styles.content}>
```
Comment (mateusz-kleszcz) [inline views/DeviceListView/components/controls/LightControls.tsx:84]: Why layout isn't applied to this component automatically 🤔

**Review by mateusz-kleszcz — CHANGES_REQUESTED** [review]
Could you align a navbar text to the left?
image

File: views/DeviceListView/components/controls/LightControls.tsx (hunk 4 above)
Comment (Phabes) [reply views/DeviceListView/components/controls/LightControls.tsx:84]: Done

**Review by mateusz-kleszcz — APPROVED** [review]

**Pull request merged**